add a doc link (#101)
OpenLiberty/guide-cors: README.adoc

|[hotspot=20 file=0]`exposeHeaders` | Headers that are safe to expose to clients. The value is set to `MyHeader`.
|===

> If you want to learn more configuration, see the https://www.openliberty.io/docs/latest/reference/config/cors.html[Configuration for the Cross-Origin Resource Sharing documentation^] 

Reviewer: ```suggestion
For more information about these and other CORS attributes, see the https://www.openliberty.io/docs/latest/reference/config/cors.html[cors element documentation^] 
```
